tools/earth/landcover_subindic.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="landcover_subindicator" name="Land cover degradation" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>subindicator to track land changes</description>
+    <macros>
+        <token name="@TOOL_VERSION@">0.1.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="1.26.3">numpy</requirement>
+        <requirement type="package" version="3.8.4">matplotlib</requirement>
+        <requirement type="package" version="1.3.10">rasterio</requirement>
+        <requirement type="package" version="2.1.14">trends_earth_algorithms</requirement>
+        <requirement type="package" version="2.1.14">trends_earth_schemas</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python '$__tool_directory__/land_cover.py'
+            --raster_1 '$raster_1'
+            --raster_2 '$raster_2'
+            --json '$json'
+    ]]></command>
+    <inputs>
+        <param name="raster_1" type="data" format="tiff" label="A raster image of the land at a reference year" />
+        <param name="raster_2" type="data" format="tiff" label="A raster image of the land at the year you want to study" />
+        <param name="json" type="data" format="json" label="A file containing the land specifications" help="You can directly use the default one here https://github.com/fair-ease/earth-critical-zone-galaxy/blob/main/landcover.json but you can also customize it"/>
```

Comment:
If people should use this default, maybe include it in the wrapper, so that people don't need to download it separately. In addition to this you can offer a history file as an option.

Reply:
Yeah that was my initial idea but Can I download a file like that put it as the default value really ? 

A history file option ? what do you mean ?

Reply:
Put this in a conditional asking "use default land specific geojson file provided my Marie" or "provide my own file".

Reply:
ah yes and if the user say yes i put the path directly and if no a normal data param

Reply:
That way is good ?